constants/appLinks.ts

```diff
@@ -49,9 +49,12 @@ const TEXTBOOK_BETA_START_LEARNING: GeneralLink = {
   }
 }
 
-const textbookBetaIntroductionCourseStartLearningUrl = 'https://learn.qiskit.org/course/introduction'
-const textbookBetaQMLStartLearningUrl = 'https://learn.qiskit.org/course/machine-learning'
-const summerSchoolCourseStartLearningUrl = 'https://learn.qiskit.org/summer-school/2021/'
+const START_LEARNING_URL = {
```

Comment:
I like this reorganisation a lot! But I think for this we should be using a [TypeScript enum](https://www.typescriptlang.org/docs/handbook/enums.html) instead.